generators/2.0/java/index.js

```diff
@@ -27,15 +30,15 @@ module.exports = class extends Generator {
                     'Gradle',
                 ]
             },
-            // {
-            //     type: 'list',
-            //     name: 'framework',
-            //     message: 'Select your framework',
-            //     choices: [
-            //         'Java (no framework)',
-            //         'SpringBoot (experimental)'
-            //     ]
-            // },
+            {
+                type: 'list',
+                name: 'framework',
+                message: 'Select your framework',
```

Comment:
If we ask for a framework and then offer no framework as a choice, is it ok ? :p (just kidding)